cluster/examples/kubernetes/nfs/operator.yaml

```diff
@@ -23,12 +23,13 @@ metadata:
   name: rook-nfs-operator
 rules:
 - apiGroups:
-  - ""
+  - "*"
```

Comment:
consider using the explicit apiGroups for storageclasses instead of introducing a wildcard here, e.g.: https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/operator.yaml#L296-L303

Reply:
can I take this in a separate PR @jbw976 as it involves only YAML changes?

Reply:
i think that we should consistently keep the RBAC defined in the operator deployments to be as explicitly scoped as possible.  This should be a pretty simple change, as shown in https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/common.yaml#L341-L348:
```
- apiGroups:
  - storage.k8s.io
  resources:
  - storageclasses
  verbs:
  - get
  - create
```

I think those are the correct verbs too, I only see instances of GET and CREATE in the new code

Reply:
done